src/Kafka/Consumer.hs

```diff
@@ -159,7 +157,7 @@ pollMessage c@(KafkaConsumer _ (KafkaConf _ qr _)) (Timeout ms) = liftIO $ do
 -- Unlike 'pollMessage' this function does not return usual "timeout" errors.
 -- An empty batch is returned when there are no messages available.
 --
--- This API is not available when 'userPolls' is set.
```

Comment:
AFAIU this was a fragment of the past coming from https://github.com/haskell-works/hw-kafka-client/pull/108